[fr] Translation of Zero Code Java docs

clementduveau opened the pull request:
Following https://github.com/open-telemetry/opentelemetry.io/pull/7351

dmathieu commented on content/fr/docs/zero-code/java/spring-boot-starter/sdk-configuration.md:
```diff
@@ -0,0 +1,284 @@
+
+Ce Spring Starter supporte les
+[métadonnées de configuration](https://docs.spring.io/spring-boot/docs/current/reference/html/configuration-metadata.html),
+ce qui signifie que vous pouvez voir et autocompléter toutes les propriétés
```
`autocompléter` to add to the list of valid words.

clementduveau commented:
Hey hey, I fixed the headers as suggested by @vitorvasc in the PHP PR (and added words in the dictionnary).

I still see errors, I will adress them but first, I have a question: I created a script to generate the headers ID. Is this something interesting to contribute back or should I just keep to myself for now ?

vitorvasc commented:
Hi @clementduveau! I've also left a comment on the other pull request about the last PHP issue that need to be fixed 😄

If you need any help with the broken checks on this PR, feel free to reach me on Slack. 

------

> I created a script to generate the headers ID. Is this something interesting to contribute back or should I just keep to myself for now ?

You're definitely welcome to share the script, we'd love to discuss more about it! I noticed there's a script pushed to this branch as well, [generate-header-ids.js](https://github.com/open-telemetry/opentelemetry.io/pull/7389/files#diff-56abfa6c4734b35c567c57462f08c6f2041610914e9abea551d0574f05a4ac17). Is it the one you're referring to? 

If you don't mind, could you move this script to a new branch and open a draft pull request for it? That will help keep things organized and make the reviews a bit easier too.

clementduveau commented:
> If you don't mind, could you move this script to a new branch 

Woops... let me revert that and fix the checks

vitorvasc reviewed (approved):
RSLGTM ✅
